src/services/ledger/index.types.d.ts

```diff
@@ -203,7 +203,7 @@ type IntraledgerLedgerMetadata =
 
 type SendLedgerMetadata = AddOnchainSendLedgerMetadata | AddLnSendLedgerMetadata
 
-type PaginationArgs = { after?: LedgerTransactionId; before?: LedgerTransactionId }
+type PaginatedResp<T> = { slice: T[]; total: number }
```

Comment:
Maybe let's come up with a better name for this. `slice` is kinda confusing because it's also a method on array. @vindard @dolcalmi ?